### DISCUSSION TITLE
Discord Server for this mod lol

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [mayo78/vsRobloxButNotReally#5] [General]
https://discord.gg/PT43rqNew7

server icon is Prism as a clown cause he fucked up a lot of shit before i kicked him off my team lol

**Comment by Theoyeah** [comment]
when i don't have discord

**Reply by ThatOneSillyGuyDon** [reply]
> when i don't have discord

Me when I can’t get on till I get back home which isn’t till fucking 5PM lol
